### PR TITLE
MM-37969 Cypress/E2E: fix tests related to file preview

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fixed tests related to file preview.

Added new UI commands for post, RHS and file preview.

Test report (updated)
- https://automation-dashboard.vercel.app/cycles/d85299f7-a571-4dc3-a01e-b0ddc02ce412 (1st run)
- https://automation-dashboard.vercel.app/cycles/1c619a17-f9cb-4797-8def-6692ea9701d8 (2nd run, with fix to file upload intercept and updated to latest master)

Screen Shot 2021-09-20 at 8 04 36 PM

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by metanerd — APPROVED** [review]
Thank you!

File: e2e/cypress/support/ui/file_preview.d.ts [hunk 1]
```diff
@@ -20,35 +20,89 @@ declare namespace Cypress {
...
+         *
          * @example
          *   cy.uiGetFileThumbnail('image.png');
          */
         uiGetFileThumbnail(filename: string): Chainable;
 
+        /**
+         * Get file upload preview located below post textbox
+         *
+         * @param {string} filename
```
Comment (josephbaylon) [inline e2e/cypress/support/ui/file_preview.d.ts:33]:
```suggestion
```

**Review by josephbaylon — APPROVED** [review]
LGTM. Just minor non-blocking comment

File: e2e/cypress/support/ui/file_preview.d.ts (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/support/ui/file_preview.d.ts]: Thanks Joe! Done.

**Comment by saturninoabril** [comment]
Done cherry-pick into release-6.0

**Pull request merged**